Formula/l/lispkit.rb

```diff
@@ -0,0 +1,69 @@
+class Lispkit < Formula
+  desc "Scheme framework for extension and scripting languages on macOS and iOS"
+  homepage "https://lisppad.app"
+  url "https://github.com/objecthub/swift-lispkit/archive/refs/tags/2.6.0.tar.gz"
+  sha256 "33ce9c6c4bd99e91308ec2a92c3850f6d63d373b8be2ec71d7e5b329fc6e394a"
+  license "Apache-2.0"
+
+  # Build dependencies
+  depends_on xcode: ["14.0", :build]
+
+  uses_from_macos "swift" => :build
```

Comment:
```suggestion
  depends_on xcode: ["14.0", :build]
  depends_on :macos
```

Reply:
Thanks for the review and the proposed changes. I followed your advice (but restricted the macOS version to Sonoma and upwards). Not sure why this still fails on Linux. Any advice is really appreciated as this executes without any problems on my machine.